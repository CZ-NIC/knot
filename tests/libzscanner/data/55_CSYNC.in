$ORIGIN	.
$TTL	1

; OK
@	CSYNC	0	0			; The simplest variant - without bitmap
@	CSYNC	4294967295 65535		; Maximal numbers
@	CSYNC	0	0	TYPE0		; Minimal type number
@	CSYNC	0	0	TYPE65535	; Maximal type number
@	CSYNC	0	0	TYPE0 A NS	; First bitmap window
@	CSYNC	0	0	TYPE0 TYPE256 TYPE512 TYPE32768	; First, second, third and 128. bitmap window
@	TYPE62	\# 6 000000000000		; TYPE + Hexadecimal rdata
@	TYPE62	0	0			; TYPE
@	csync	0	0			; Type in lower-case

; KO
@	CSYNC
@	CSYNC					; Empty rdata
@	CSYNC	\# 0				; Hex empty rdata
@	CSYNC	4294967296 0			; Serial overflow
@	CSYNC	0	65536			; Flags overflow
@	CSYNC	0	0	TYPE65536	; Type number overflow
@	CSYNC	0	0	X		; Unknown type
