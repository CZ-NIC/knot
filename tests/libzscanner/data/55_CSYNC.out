OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003E
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003E
RDATA=FFFFFFFFFFFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003E
RDATA=000000000000000180
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003E
RDATA=000000000000FF200000000000000000000000000000000000000000000000000000000000000001
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003E
RDATA=0000000000000001E0
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003E
RDATA=000000000000000180010180020180800180
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003E
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003E
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003E
RDATA=000000000000
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_NUMBER32_OVERFLOW
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_BAD_BITMAP
------
